lib/generators/shopify_app/user_model/templates/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  include ShopifyApp::SessionStorage
```

Comment:
Why use a single session storage mixin rather than a different mixin for each Shop or User? Is seems needlessly abstract.

Reply:
@tanema This was for backwards compatibility - any existing apps that use the `shopify_app` gem will have had the `shop.rb` file copied to their app. This `Shop` model extends `ShopifyApp::SessionStorage`.

One of my goals was to have a drop-in replacement that would not require users of the library to make changes to their existing code in order to upgrade so as to keep per-user tokens strictly a feature addition.

Happy to revisit this decision though.